content/en/actions/connections/_index.md

```diff
@@ -173,14 +173,14 @@ To create a connection group:
 
 To use a connection group:
 
-1. In your workflow, select an action that requires a connection.
+1. In your workflow or app, select an action that requires a connection.
 1. In the **Connection** field, in the drop-down, select the desired connection group under **Groups**.
 1. Fill in the desired values for the connection group **Identifiers**. For example, if your connection group is defined using the `env` Identifier Tag, and you have two environments, `prod` and `staging`, you could use either of those values (or an expression that evaluates to one of those values).
-1. Fill in any other required step values, then click **Save** to save your workflow.
+1. Fill in any other required step values, then click **Save**.
 
-**Note**: You can only use connections within a group if you have [Resolver permission][12] for those connections. If a workflow tries to use a connection you don't have Resolver permission for, the workflow fails with a `403 Forbidden` error. To fix this issue, you can:
-- Configure your workflow so that it can't point to a connection you don't have Resolver permission for.
-- Remove the connection for which you don't have Resolver permission from the connection group. <div class="alert alert-danger"><strong>Note</strong>: If you are using a connection group for multiple workflows, removing a connection that another workflow relies on causes that workflow to fail.</div>
+**Note**: You can only use connections within a group if you have [Resolver permission][12] for those connections. If a workflow or app tries to use a connection you don't have Resolver permission for, it fails with a `403 Forbidden` error. To fix this issue, you can:
+- Configure the workflow or app so that it can't point to a connection you don't have Resolver permission for.
+- Remove the connection for which you don't have Resolver permission from the connection group. <div class="alert alert-danger"><strong>Note</strong>: If you are using a connection group for multiple workflows or multiple apps, removing a connection that another workflow relies on causes that workflow to fail.</div>
```

Comment:
```suggestion
- Configure the workflow or app so that it can't point to a connection that doesn't have a Resolver permission.
- Remove the connection that doesn't have a Resolver permission from the connection group. <div class="alert alert-danger"><strong>Note</strong>: If you are using a connection group for multiple workflows or multiple apps, removing a connection that another workflow relies on causes that workflow to fail.</div>
```